astropy/coordinates/matrix_utilities.py

```diff
@@ -35,6 +35,10 @@ def rotation_matrix(angle, axis="z", unit=None):
     """
     Generate matrices for rotation by some angle around some axis.
 
+    .. versionadded:: 7.0.0
+        This function was made available from the
+        `astropy.coordinates` namespace.
```

Comment:
I don't think this is correct actually.
This is refering to https://github.com/astropy/astropy/pull/18418, which is currently unreleased, but is expected to land in 7.2.0.